<?xml version="1.0"?>
<launch>
  <!-- Arguments -->
  <arg name="param_file" default="$(find gym_gazebo)/param/param.yaml"
    doc="Parameter file"/>
  <arg name="log_path" default="$(find gym_gazebo)/logs"
    doc="Log file path"/>
  <arg name="model_dir" default="$(find gym_gazebo)/models"
    doc="Log file directory"/>

  <!-- Train node -->
  <node pkg="gym_gazebo" type="sbtraining.py" name="sbtrain" output="screen">
    <rosparam file="$(arg param_file)"/>
    <param name="log_path" value="$(arg log_path)"/>
    <param name="model_dir" value="$(arg model_dir)"/>
  </node>
</launch>
